calcom/cal.com
# fix: Only enforce min seats for orgs

## What does this PR do?



This PR fixes the logic to only change team subscription amounts for orgs and not normal teams

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Enable team billing
- Create a team and invite members
- Delete some members
- On Stripe the subscription count should be updated


## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
